wikitools/link_checker.py

```diff
@@ -0,0 +1,100 @@
+import os
+import typing
+
+from wikitools import redirect_parser, reference_parser, errors, link_parser, article_parser
+
+
+class DetailedError(typing.NamedTuple):
```

Comment:
nesting small wrappers like this only makes the [code where it's used](https://github.com/Walavouchey/osu-wiki-tools/pull/1/files#diff-878ee5028b9a35bc6a9f44d24e14468d5a7fa579670022c27feaa9358c94a97bR103-R124) more annoying to read. this should also include the line number, or maybe poke everything into `LinkError()`

Reply:
done